tests/testthat/test-financial_functions.R

```diff
@@ -13,10 +13,13 @@ test_that("calculate_net_profits penalizes companies for late build out of low
 
 
   test_shock_year <- 2021
+  test_market_passthrough <- 0
```

Comment:
It would make sense to also add a test where test_market_passthrough is 1. You could then compare the results to those that occured with a market_passthrough of 0 :)

Reply:
done